src/App.js

```diff
@@ -7,7 +7,7 @@ function App() {
     <div className="App">
       <header className="App-header">
         <p>
-         I can use Giithub!
+         Learning Github...
         </p>
         <a
           className="App-link"
```

Comment:
CSS classnames should not be capitalized "App-link" should be "app-link". Same with other classnames on this page. Update and then i'll approve!

Reply:
got it